### DISCUSSION TITLE
Several Questions

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [dailydotdev/daily#851] [General]
1st Question:
I recently submitted a new blog post which was later rejected since the post was already on daily.dev (as per the email I received). However, the thing to note is that the post did not show before submitting, and thus the daily limitation of 3 was reached. Is this a bug? I submitted a post, and it was counted even though it already existed. Here are some screenshots for reference:

<img src="https://user-images.githubusercontent.com/74038190/226993702-66942176-b537-4197-b5a8-6e35b07aa8bc.png" width="800" />
<img src="https://user-images.githubusercontent.com/74038190/226993710-b7f33b10-4a3b-4d21-b516-08409df65f00.png" width="800" />

<br>

2nd Question:
Suppose I get two upvotes from people who haven't crossed the 250 reputation threshold. In this case, the reputation increase will not be counted. However, if these individuals are likely to cross the threshold in the future, will my reputation increase too?

<br>

3rd Question:
May I know the typical response time for emails sent to the daily.dev team? I haven't received a reply yet, and it has been a day.

I have reported several posts according to the guidelines that I found were not up to the standards since then, but I haven't received any recognition, even when the post was removed. I have sent the reference images in the mail.

<br>

4th Question:
I have suggested over 12 new sources, but I haven't received any update or response from you. It has been more than three days, and I would appreciate an update on the status of my suggestions.

<br>

Suggestion
I have a suggestion. It might be better to consider people who spend half the time on a blog post as having read it. Sometimes, we open a post and lose interest after a few lines, so I wouldn't want a tag indicating that I've read the post if I didn't genuinely do so.

**Comment by idoshamun** [comment]
Hi @Anmol-Baranwal,

I will reply to you via email. Usually it takes a few days to reply to emails and up to two weeks to review source suggestions

**Reply by Anmol-Baranwal** [reply]
@idoshamun 
I have only emailed 3rd and 4th question. Please answer 1st & 2nd.

**Reply by idoshamun** [reply]
1. This is not a bug but an intentional feature. We want to limit the number of submissions regarding whether they succeed or not.
2. No, you will not get reputation points retroactively. You get reputation points from upvotes of users with over 250 in the moment of the action.
I hope this makes sense.

**Reply by Anmol-Baranwal** [reply]
Thanks for letting me know.
